Mathlib/Algebra/BigOperators/Fin.lean

```diff
@@ -388,6 +388,17 @@ theorem finPiFinEquiv_single {m : ℕ} {n : Fin m → ℕ} [∀ i, NeZero (n i)]
   rintro x hx
   rw [Pi.single_eq_of_ne hx, Fin.val_zero', zero_mul]
 
+/-- Equivalence between the Sigma type `(i : Fin m) × Fin (n i)` and `Fin (∑ i : Fin m, n i)`. -/
+def finSigmaFinEquiv {m : ℕ} {n : Fin m → ℕ} : (i : Fin m) × Fin (n i) ≃ Fin (∑ i : Fin m, n i) :=
+  match m with
+  | 0 => @Equiv.equivOfIsEmpty _ _ _ (by simp; exact Fin.isEmpty')
```

Comment:
Do you need `@` here?

Reply:
Yes, the one argument I'm providing (and it seems necessary to manually provide) is an instance parameter which doesn't have name. An alternative would be to write
```
let _ : IsEmpty (Fin (∑ i, n i)) := by simpa using Fin.isEmpty'
Equiv.equivOfIsEmpty _ _
```
which works, if you think that's cleaner.